feature-dapp-impl/src/main/java/io/novafoundation/nova/feature_dapp_impl/presentation/search/DAppSearchCommunicator.kt

```diff
@@ -12,6 +12,15 @@ interface DAppSearchResponder : InterScreenResponder<SearchPayload, Response>
 
 interface DAppSearchCommunicator : DAppSearchRequester, DAppSearchResponder {
 
-    @Parcelize
-    class Response(val newUrl: String?) : Parcelable
+    sealed interface Response : Parcelable {
+
+        @Parcelize
+        object TabChanged : Response
```

Comment:
I dont see this is handled anywhere - do we even need it?

Reply:
We need to handle only new url, but I anyway send this response